eql/parser.py

```diff
@@ -1162,7 +1184,12 @@ def sequence(self, node):
         if node['with_params']:
             params = self.time_range(node['with_params']['time_range'])
 
-        queries, close = self._get_subqueries_and_close(node, allow_fork=True)
+        allow_runs = self._elasticsearch_syntax
+
+        queries, close = self._get_subqueries_and_close(node, allow_fork=True, allow_runs=allow_runs)
+        if len(queries) == 1 and not self._elasticsearch_syntax:
```

Comment:
Should this _always_ be an error, regardless of specific syntax?

Reply:
yep. sequence should always be 2+.
syntax error for endgame mode, semantic error for elastic mode